shared/source/os_interface/windows/driver_info_windows.cpp

```diff
@@ -34,20 +34,30 @@ std::string getCurrentLibraryPath() {
 
 namespace NEO {
 
-DriverInfo *DriverInfo::create(const HardwareInfo *hwInfo, OSInterface *osInterface) {
-    if (osInterface) {
-        auto wddm = osInterface->get()->getWddm();
-        DEBUG_BREAK_IF(wddm == nullptr);
+DriverInfo *DriverInfo::create(const HardwareInfo *hwInfo, const OSInterface *osInterface) {
+    if (osInterface == nullptr) {
+        return nullptr;
+    }
+
+    auto osInterfaceImpl = osInterface->get();
+    if (osInterfaceImpl == nullptr) {
+        DEBUG_BREAK_IF(true);
+        return nullptr;
+    }
 
-        std::string path(wddm->getDeviceRegistryPath());
-        return new DriverInfoWindows(std::move(path));
+    auto wddm = osInterfaceImpl->getWddm();
+    if (wddm == nullptr) {
+        DEBUG_BREAK_IF(true);
+        return nullptr;
     }
```

Comment:
I believe it should never happen and if it does we just need to bail out by using UNRECOVERABLE_IF

Reply:
The check has been replaced with `UNRECOVERABLE_IF()`.